### PR TITLE
Add annotation player

### PR DESCRIPTION
**Pull request by glmdgrielson**
This addresses issue #110. Yay for adding annotations back!

This should do until the archive system gets revamped.

### CODE REVIEW

File: src/invidious/views/components/player.ecr [hunk 1]
```diff
@@ -167,7 +167,6 @@ if (bpb) {
 <% end %>
 
 
-var video = player;
```
Comment (omarroth) [inline src/invidious/views/components/player.ecr]: `videojs-youtube-annotatations.js` relies on the player being called `video`. Without, it throws:
```
video is not defined
```
[here](https://github.com/afrmtbl/videojs-youtube-annotations/blob/master/dist/dist.js#L408).

Comment (glmdgrielson) [reply src/invidious/views/components/player.ecr]: Oh... oops!

**Comment by omarroth** [comment]
I'll plan on adding an updated version of `videojs-youtube-annotations.js` before merging, since there's quite a few features missing that are included in the extension it's based on. 

The `/api/v1/annotations/:id` is temporary and will also change once everything has been uploaded to the Internet Archive.

https://dev.invidio.us is now pulling from this branch, so if annotations for a video are available from the temporary API, you should be able to view them.

File: src/invidious/views/components/player.ecr (hunk 1 above)
Comment (omarroth) [reply src/invidious/views/components/player.ecr]: Updated dist.js with a fixed copy.

**Comment by glmdgrielson** [comment]
So the `video` issue gas been resolved? What about the 4.3 one?

**Comment by omarroth** [comment]
I expect that will be fixed in [afrmtbl/annotationlib](https://github.com/afrmtbl/annotationlib).

**Comment by omarroth** [comment]
`<%= video.id %>` refers to the current video. For annotations, we want to find the video that's being linked, so we use the video ID included in the search parameters.

**Comment by AgentGrady** [comment]
Several videos that had working annotations on invidious just a few days ago have suddenly stopped working. Why is this?

**Comment by omarroth** [comment]
Mind posting the output of the console or linking to an example?

**Comment by AgentGrady** [comment]
Ok, here are some examples

https://dev.invidio.us/watch?v=29-q7YnyUmY

https://dev.invidio.us/watch?v=FBeuzMM7HSc

https://dev.invidio.us/watch?v=gf5YT60DaqQ

**Comment by AgentGrady** [comment]
Annotations have suddenly stopped working on several videos.

**Comment by AgentGrady** [comment]
https://dev.invidio.us/watch?v=FBeuzMM7HSc&iv_load_policy=1

**Pull request merged**